Source/Resource/Translation/apd_ko_KR.ts

```diff
@@ -5,256 +5,272 @@
     </message>
     <message>
         <source>No paired device found.
 You need to pair your AirPods in Windows Bluetooth Settings first.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">페어링된 장치를 찾을수 없습니다.
+먼저 윈도우 블루투스 설정에서, AirPod과 페어링 해주세요.</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <source>It fixes short audio playback problems, but may increase battery consumption.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">짧은 오디오 재생 문제를 해결하지만 배터리 소모를 증가 될수 있습니다.</translation>
     </message>
     <message>
         <source>It automatically pauses or resumes media when your AirPods are taken out or put in your ears.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">에어팟을 빼거나 귀에 넣으면 자동으로 미디어를 일시중지하거나 재생합니다.</translation>
     </message>
     <message>
         <source>Unavailable</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">사용불가</translation>
     </message>
     <message>
         <source>Disconnected</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">연결끊김</translation>
     </message>
     <message>
         <source>Waiting for Binding</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">연결 대기중/translation>
     </message>
 </context>
 <context>
     <name>SelectWindow</name>
     <message>
         <source>Select</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">선택</translation>
     </message>
 </context>
 <context>
     <name>SettingsWindow</name>
     <message>
         <source>AirPodsDesktop Settings</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">AirPodsDesktop 설정</translation>
     </message>
     <message>
         <source>General</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">기본</translation>
     </message>
     <message>
         <source>Open logs directory</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">로그파일 열기</translation>
     </message>
     <message>
         <source>Bluetooth maximum receiving range</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">블루투스 최대 수신범위</translation>
     </message>
     <message>
         <source>Unbind AirPods</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">AirPods 연결해제</translation>
     </message>
     <message>
-        <source>Launch when system starts</source>
+        <source>시스템 시작 시 실행</source>
         <translation type="unfinished"></translation>
     </message>
```

Comment:
Here is a mistake, the source text should not be modified.

Reply:
ok thank

Reply:
sorry. I have installed Qt Linguist and am fixing it again.

Reply:
@lu1z5ec There are no new changes here, you should commit your changes.